ssl-config-core/src/main/scala/com/typesafe/sslconfig/ssl/Config.scala

```diff
@@ -139,25 +137,24 @@ object TrustStoreConfig {
     apply(Option(data.orElse(null)), Option(filePath.orElse(null)))
 }
 
-
 /**
  * The key manager config.
  *
  * @param algorithm The algoritm to use.
  * @param keyStoreConfigs The key stores to use.
  */
-final class KeyManagerConfig private[sslconfig](
-  val algorithm: String = KeyManagerFactory.getDefaultAlgorithm,
-  val keyStoreConfigs: immutable.Seq[KeyStoreConfig] = Nil) {
+final class KeyManagerConfig private[sslconfig] (
+    val algorithm:       String                        = KeyManagerFactory.getDefaultAlgorithm,
+    val keyStoreConfigs: immutable.Seq[KeyStoreConfig] = Nil) {
 
   def withAlgorithm(value: String): KeyManagerConfig = copy(algorithm = value)
   def withKeyStoreConfigs(value: scala.collection.immutable.Seq[com.typesafe.sslconfig.ssl.KeyStoreConfig]): KeyManagerConfig = copy(keyStoreConfigs = value)
 
   private def copy(
-    algorithm: String = algorithm,
+    algorithm:       String                                                                    = algorithm,
```

Comment:
Hum... looks like [Scalariform's `alignparameters`](https://github.com/scala-ide/scalariform#alignparameters) is behaving differently now. Maybe we should change the settings to use the default (`false`)?

Reply:
@marcospereira should be good to go now